src/sentry/static/sentry/app/views/settings/organization/general/organizationGeneralSettingsView.jsx

```diff
@@ -10,9 +10,16 @@ import {
 } from '../../../../actionCreators/organizations';
 import {t, tct} from '../../../../locale';
 import ApiMixin from '../../../../mixins/apiMixin';
+import FormFieldWrapper from '../../components/forms/formField/formFieldWrapper';
```

Comment:
Eyesore, but I think consistency is better, at least that's what I'm telling myself